psi4/driver/procrouting/proc.py

```diff
@@ -25,7 +25,6 @@
 #
 # @END LICENSE
 #
-
```

Comment:
Hmm, @loriab Do we have outstanding changes to `proc.py`? We might need to revert these changes so that this file is still mergeable later.